### PR TITLE
Import and export ActionType from dependency fusion-rpc-redux ^1.4.0

### PR DESCRIPTION
**Pull request by hugoe29**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {
   RPCHandlersToken,
   ResponseError,
 } from 'fusion-plugin-rpc';
-import {createRPCReducer} from 'fusion-rpc-redux';
+import {createRPCReducer, ActionType} from 'fusion-rpc-redux';
```
Comment (lhorie) [inline src/index.js]: This should be `import type {ActionType}`

Comment (hugoe29) [reply src/index.js]: Done

**Comment by lhorie** [comment]
Looks good to me! Thanks @hugoe29 !

**Comment by lhorie** [comment]
Oh, the export also needed to be changed. I went ahead and made the adjustment

**Comment by hugoe29** [comment]
Thanks @lhorie 
It seems it is just waiting for the reviewer approval.

**Pull request merged**